### PR TITLE
Implement appr and helm install chart methods

### PR DESCRIPTION
**Pull request by fgimenez**
Towards https://github.com/giantswarm/giantswarm/issues/1902

Still missing e2e test, will come next.

### CODE REVIEW

File: service/chartconfig/v1/appr/appr.go [hunk 1]
```diff
@@ -11,10 +11,12 @@ import (
 
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
 )
 
 // Config represents the configuration used to create a appr client.
 type Config struct {
+	Fs     afero.Fs
```
Comment (xh3b4sd) [inline service/chartconfig/v1/appr/appr.go]: This is nice.

File: service/chartconfig/v1/appr/appr.go [hunk 2]
```diff
@@ -84,10 +91,26 @@ func (c *Client) GetReleaseVersion(name, channel string) (string, error) {
...
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+
+	chartTarball, err := c.doFile(req)
+	if err != nil {
+		return "", microerror.Mask(err)
+	}
+
+	return chartTarball, nil
```
Comment (xh3b4sd) [inline service/chartconfig/v1/appr/appr.go]: The var indicates it is the tarball itself but it is only a file path.

File: service/chartconfig/v1/helm/helm_test.go [hunk 3]
```diff
@@ -200,3 +200,63 @@ func Test_GetReleaseHistory(t *testing.T) {
...
+
+	for _, tc := range testCases {
+		t.Run(tc.description, func(t *testing.T) {
+			helm := Client{
+				helmClient: &helmclient.FakeClient{
+					Rels: tc.releases,
+				},
+				logger: microloggertest.New(),
+			}
+			// helm fake client does not actually use the tarball
```
Comment (rossf7) [inline service/chartconfig/v1/helm/helm_test.go]: Nitpick but this should be a sentence with a `.`.

File: service/chartconfig/v1/appr/appr.go (hunk 2 above)
Comment (fgimenez) [reply service/chartconfig/v1/appr/appr.go]: Good point, renamed to make clear it's a path.

File: service/chartconfig/v1/helm/helm_test.go (hunk 3 above)
Comment (fgimenez) [reply service/chartconfig/v1/helm/helm_test.go]: Sure, done.

**Pull request merged**